code/datums/jobs.dm

```diff
@@ -1470,8 +1468,9 @@ ABSTRACT_TYPE(/datum/job/civilian)
 
 		var/obj/item/storage/briefcase/B = M.find_type_in_hand(/obj/item/storage/briefcase)
 		if (B && istype(B))
-			new /obj/item/instrument/whistle(B)
+			B.storage.add_contents(new /obj/item/instrument/whistle(B))
```

Comment:
Yeah, add_contents() determines the end result. There are a lot of cases of stuff like this, if I remember right I didn't want to change them in case an object happens to do something extra with the passed in arg for any reason, but I'll refactor all of them if you'd like

Reply:
Yeah, I'd prefer if they were changed, as long as it isn't too much trouble. The first arg to `new` should always be the loc, and pretty much every atom/movable should function with that arg being `null`
(keyword SHOULD, but any resulting bugs are badcode and not your fault)

Reply:
https://github.com/goonstation/goonstation/pull/13023/commits/5724df86dfdc47c91b7ed36362bdbf490ba31128